var/spack/repos/builtin/packages/dune-common/package.py

```diff
@@ -0,0 +1,27 @@
+# copyright 2013-2023 lawrence livermore national security, llc and other
+# spack project developers. see the top-level copyright file for details.
+#
+# spdx-license-identifier: (apache-2.0 or mit)
+
+
+from spack.package import *
+
+
+class dunecommon(pythonpackage):
```

Comment:
The case is wrong for the class and inherited package.  See https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-distro/package.py for an example.